modules/product/index.js

```diff
@@ -31,6 +48,29 @@ module.exports = {
       }
     }
   },
+  options: {
+    filters: [
+      {
+        name: 'color',
+        label: 'Color',
+        inputType: 'select',
+        choices: [
```

Comment:
Apos yelled at me for not explicitly setting choices here. Down the road it would be nice if it found those itself.